### PR TITLE
lidar 3d buildings admin, form, x-large supplemental

### PR DESCRIPTION
**Pull request by jwhaney**
added lidar_buildings_url to collections table in lcd.models; admin.py and forms.py changes for new lidar_buildings_url for normal collection form and for x-large supplemental form in master resources; etc.

closes issue #34

### CODE REVIEW

**Comment by jwhaney** [comment]
fyi, not a big deal, but i noticed once the .zip is uploaded to the correct collection assets subdirectory in the data.tnris.org bucket, -lidar-breaklines.zip and -lidar-buildings.zip have an extra 'lidar' in the filename. guessing I can just alter that in the forms.py on lines 301 & 302? get rid of 'lidar' and just have '-breaklines.zip' & '-buildings.zip'?

**Comment by jwhaney** [comment]
prob shouldn't mess with that right now though...

**Comment by adambreznicky** [comment]
the extra lidar in the name is correct. it's not an error.

the standard naming convention uses the collection name in conjunction with the field name. so a collection with the word "lidar" in the name will have 2 instances of the word for a supplemental 'lidar breaklines' or 'lidar buildings'. it would feel weird to deviate from this simple standard to me.

example: "east coast lidar" would have a download called "east-coast-lidar-lidar-breaklines.zip" since field "lidar breaklines" is appended to the end. A collection called "Northern Lidar Elevation" would be "northern-lidar-elevation-lidar-breaklines.zip"

because also, technically, any collection (lidar or not) could have the lidar breaklines or lidar buildings upload, i dont think we should change it. also since collection names can be changed at any point, special handling to try and manage this just doesn't seem worth it to me. 

there is also the point that they have changed the standard naming convention for collections a couple times since i've been in the basement. tomorrow they may change it again, and decide the word Lidar doesn't have to be in collection names. or that it should be first, before the area descriptor.

so in short, i don't see a problem with these circumstances. what are y'alls thoughts?

**Comment by jwhaney** [comment]
> the extra lidar in the name is correct. it's not an error.
> 
> the standard naming convention uses the collection name in conjunction with the field name. so a collection with the word "lidar" in the name will have 2 instances of the word for a supplemental 'lidar breaklines' or 'lidar buildings'. it would feel weird to deviate from this simple standard to me.
> 
> example: "east coast lidar" would have a download called "east-coast-lidar-lidar-breaklines.zip" since field "lidar breaklines" is appended to the end. A collection called "Northern Lidar Elevation" would be "northern-lidar-elevation-lidar-breaklines.zip"
> 
> because also, technically, any collection (lidar or not) could have the lidar breaklines or lidar buildings upload, i dont think we should change it. also since collection names can be changed at any point, special handling to try and manage this just doesn't seem worth it to me.
> 
> there is also the point that they have changed the standard naming convention for collections a couple times since i've been in the basement. tomorrow they may change it again, and decide the word Lidar doesn't have to be in collection names. or that it should be first, before the area descriptor.
> 
> so in short, i don't see a problem with these circumstances. what are y'alls thoughts?

this make sense. thanks for clarifying.
